tdrs-frontend/src/components/FileUpload/FileUpload.jsx

```diff
@@ -7,35 +7,61 @@ import {
   clearFile,
   SET_FILE_ERROR,
   upload,
+  download,
 } from '../../actions/reports'
+
+import Button from '../Button'
+
 import createFileInputErrorState from '../../utils/createFileInputErrorState'
+import { handlePreview, getTargetClassName } from './utils'
 
 const INVALID_FILE_ERROR =
   'We can’t process that file format. Please provide a plain text file.'
 
 function FileUpload({ section, setLocalAlertState }) {
   // e.g. 'Aggregate Case Data' => 'aggregate-case-data'
   // The set of uploaded files in our Redux state
-  const files = useSelector((state) => state.reports.files)
+  const { files, year, downloadedFile } = useSelector((state) => state.reports)
```

Comment:
```suggestion
  const { files, year } = useSelector((state) => state.reports)
```
downloadedFile doesn't appear to be used